vscode_extension/src/commands/toggleUntypedCodeHighlighting.ts

```diff
@@ -1,15 +1,26 @@
-import { TrackUntyped } from "../config";
+import { QuickPickItem, window } from "vscode";
+import { TrackUntyped, SorbetExtensionConfig } from "../config";
 import { Log } from "../log";
 import { SorbetExtensionContext } from "../sorbetExtensionContext";
 
-function toggle(log: Log, trackWhere: TrackUntyped): TrackUntyped {
-  switch (trackWhere) {
+export interface TrackUntypedQuickPickItem extends QuickPickItem {
+  trackWhere: TrackUntyped;
+}
+
+function toggle(log: Log, configuration: SorbetExtensionConfig): TrackUntyped {
+  if (configuration.oldHighlightUntyped != null) {
+    return configuration.oldHighlightUntyped;
+  }
+
+  switch (configuration.highlightUntyped) {
     case "nowhere":
       return "everywhere";
+    case "everywhere-but-tests":
+      return "nowhere";
```

Comment:
Nit: toggling a tri-state flag leads to a funny construct like this. - e.g. if you toggle `everywhere-but-tests` you go to `nowhere` but if you toggle again you go to `everywhere` so `everywhere-but-tests` is not accessible.  Should `toggle` loop `nowhere` » `everywhere-but-tests` » `everywhere` » `nowhere` ?

Reply:
@damolina-stripe I believe that in your case, I've added an extra field to store "the setting before the current setting" so that toggling works. Allowing people to toggle between "everywhere" and "everywhere-but-tests" was a central focus of this change, and I've verified that it works in my editor locally.

Reply:
Or perhaps, this not longer is a `Toggle` and should be a `Configure Code Highlight`  action.

Reply:
There is already a separate "configure" action implemented below, which allows you to choose from a drop down.

See the `oldHighlightUntyped` field access immediately above for how the sticky toggle works.